src/Jackett.Common/Definitions/uhdbits.yml

```
@@ -174,5 +188,5 @@ search:
           args: ["(?i)\\bSeason (\\d+)\\b", "S$1"]
         - name: trim
         - name: append
-          args: " {{ .Result._release_info }}"
+          args: " {{ if eq .Result._release_is_music \"1\" }}{{ .Result._release_info_music }}{{ else }}{{ .Result._release_info }}{{ end }}"
```

```suggestion
          args: " {{ if eq .Result.category \"2\" }}{{ .Result._release_info_music }}{{ else }}{{ .Result._release_info }}{{ end }}"
```

Does this work? And then just get rid of `_release_is_music`?